panopticum/models.py

```diff
@@ -472,10 +472,9 @@ class ComponentVersionModel(models.Model):
     qa_upgrade_tests_signoff = panopticum.fields.SigneeField(related_name='signed_upgrade_tests')
 
     # meta
-
-    meta_update_by = models.ForeignKey(User, on_delete=models.PROTECT, blank=True, null=True, related_name='updater_of')
-    meta_update_date = models.DateTimeField(db_index=True)
-    meta_deleted = models.BooleanField(default=False)
+    release_date = models.DateField(null=True, blank=True)
```

Comment:
Why the prefix has been removed?

meta_* prefix meant we do not want to make these fields available for manual edit and it was an easy way not to forget to update it on save.

Reply:
Models is low-level layer. We should not care about frontend level here. Today we wont not present field. Tomorrow we can want to present at frontend\admin panel that field. Django models it's about how to get data, not how to present data.
Anyway I will remove that field and switch sorting by version name at beta milestone. When we will discuss and accept any solution about getting latest version may be we revert that field or something else.

Reply:
The "meta_*" fields are not about view and frontend.

meta_* fields populated automatically on save and must not be managed by user